### ISSUE TITLE
The performance shown in the paper cannot be reproduced

### ISSUE THREAD

**Issue by huyu398** [kleimerTU/HumanCentricLayouts#3]
@kleimerTU

Thank you for sharing the code.
I have tried to reproduce the performance shown in the paper.
The paper shows the trends in the loss function as shown in the figure below.
![image](https://user-images.githubusercontent.com/4333569/216942249-000d15aa-0d45-4eb3-aa3d-5a24945d95a2.png)

I built an execution environment using Docker as shown in the repository below and re-trained the models in each ablation.
https://github.com/huyu398/HumanCentricLayouts
A visualization of the loss trends as a result of retraining is shown in the figure below.
![loss](https://user-images.githubusercontent.com/4333569/216948797-d032043d-3de2-47ba-8c9f-aaa5a8e13147.png)
Obviously, the loss values are different from the paper's performance, and I would like to determine the cause of this.

**Are you sure that the code you are sharing is capable of reproducing the performance shown in the paper?
Please give us your opinion as to what could be the reason why the performance is not reproduced.**

The latest data ([refined 3D-FRONT dataset](https://tianchi.aliyun.com/dataset/dataDetail?dataId=65347)) from [3D-FRONT](https://tianchi.aliyun.com/specials/promotion/alibaba-3d-scene-dataset) was used for the training data.

Thanks for reading. I would appreciate your response.

**Comment by kleimerTU** [comment]
Hi, 

The loss functions from the paper are based on a model that uses the binary mask of the room shape as conditioning, while I think yours is trained without conditioning, is that correct? It makes sense that the plots look different then.

Second, can you maybe check if your losses are all plotted in the correct order? I have never seen the baseline validation loss being the highest in any experiment, usually it is the lowest. If the order of the validation losses was switched, it would look similar to what I'm usually seeing for unconditioned models (same for the training loss). The ergonomic losses look fine to me.

**Comment by huyu398** [comment]
@kleimerTU 
Thank you for your reply.

Oh sorry, the loss function illustrated was trained without conditioning on the binary mask.
I'll try again with conditioning, thanks for your advice.

The plotted order should be correct.
Tried several times, the baseline model has the highest loss in my replications.
I'll check this well, too.

Thank you for reading!

**Comment by WHQCHINA** [comment]
@kleimerTU @huyu398 
Hello everyone, when I was training the model, from [3D-FRONT](https://tianchi.aliyun.com/specials/promotion/alibaba-3d-scene-dataset) was used for the training , the Ergonomic loss value fluctuated significantly, as shown in the following two pictures:
The first image is a pre-trained model trained using all types of room data, and the Ergonomic loss of the validation set is abnormal：
![loss](https://github.com/kleimerTU/HumanCentricLayouts/assets/10349500/6ada5964-b52a-47d7-bb31-ff3f304260dc)
The model configuration parameters are as shown below：
<img width="280" alt="image" src="https://github.com/kleimerTU/HumanCentricLayouts/assets/10349500/7cc0ee7a-2d46-4d9d-bf0b-737f08ad7629">

The second image is a fine-tuned livingrooms based on the pre-trained model, and the Ergonomic loss changes little between the training set and the validation set.
![loss-1](https://github.com/kleimerTU/HumanCentricLayouts/assets/10349500/f0974839-7bde-49bd-acc4-72902dd7191a)
The model configuration parameters are as shown below：
<img width="469" alt="image" src="https://github.com/kleimerTU/HumanCentricLayouts/assets/10349500/94513e97-f105-47b1-ac55-6ac43c7afb34">


At the same time, train loss and val loss do not look as good as in the paper. I am not sure where  the problem is. I would like to ask for help.
